src/context/data/FilterContext.js

```diff
@@ -0,0 +1,23 @@
+import { createContext, useContext, useReducer } from "react";
+import { filterReducer } from "../../reducer/filterReducer";
+
+const FilterContext = createContext(null);
+
+const FilterProvider = ({ children }) => {
+  const [state, dispatch] = useReducer(filterReducer, {
```

Comment:
Since we may have multiple context you can have better naming convention like ```filterState``` or ```filterDispatch```

Reply:
sure kevin will do that thanks for suggestion.